src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -400,25 +403,23 @@ public StudyEntity doGetStudyWithPreFetchedLoadFlowResultAndUpdateIsPrivate(UUID
                 .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    private Mono<? extends Throwable> handleStudyCreationError(String studyName, ClientResponse clientResponse) {
+    private Mono<? extends Throwable> handleStudyCreationError(UUID studyUuid, String studyName, String userId, ClientResponse clientResponse) {
         return clientResponse.bodyToMono(String.class).flatMap(body -> {
             try {
-                String message;
                 JsonNode node = new ObjectMapper().readTree(body).path("message");
                 if (!node.isMissingNode()) {
-                    message = node.asText();
-                    emitStudyError(studyName, UPDATE_TYPE_STUDIES, message);
+                    emitStudyCreationError(studyUuid, studyName, userId, true, node.asText());
```

Comment:
if this is force to true here, then it should not be called "publicstudy", it should be called "broadcastnotification" or something like that.
But instead, I think we should send the real value of isPrivate, and add filter in the notification-server

Reply:
Done